### DISCUSSION TITLE
how to make a Overlay

### DISCUSSION THREAD

**Discussion by arthurmelton** [tauri-apps/tauri#4810] [Q&A] [answered]
So I was wondering if there is a built in way to make a overlay (like discords overlay on games). I could do this manually by getting the pid of a task, get the position of the window from a pid, then do [set_position](https://docs.rs/tauri/latest/tauri/window/struct.Window.html#method.set_position) on it and update a few times every second or even every frame. I was just wondering if there is a easier way of doing it, or if I will have to do this.

**Comment by abisar** [comment]
Have you tried [set_always_on_top](https://docs.rs/tauri/latest/tauri/window/struct.Window.html#method.set_always_on_top)?

**Reply by arthurmelton** [reply]
Yes, I was mainly trying to see if there was a easy way to snap the window to another window.

**Reply by Lixuhuilll** [reply]
It doesn't work for exclusive full-screen applications.

**Comment by arthurmelton** [comment, marked as answer]
I think the way I suggested of snapping the window to another is the best.

**Reply by Rhydderch** [reply]
Were you able to achieve an overaly window in Tauri? Could you share a screenshot of the result? :)

**Reply by MilesRayne** [reply]
I'd also be interested in seeing a working implementation of a Tauri App window overlayed over an exclusive full-screen application.
